bitnami/elasticsearch/values.yaml

```
@@ -298,6 +298,10 @@ master:
   ## Note: podAffinityPreset, podAntiAffinityPreset, and  nodeAffinityPreset will be ignored when it's set
   ##
   affinity: {}
+  ## @param master.priorityClassName Priority Class Name
```

I would be more specific in the description, like:

```suggestion
  ## @param master.priorityClassName Master pods Priority Class Name
```

The same applies to the rest. Also, please double check the text is the same as in the README file.